android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -673,6 +643,22 @@ private void scheduleNextNotificationIfRepeating(Bundle bundle) {
         }
     }
 
+    private int getRepeatField(String repeatType) {
+        switch (repeatType) {
+            case "month":
+                return Calendar.MONTH;
+            case "week":
+                return Calendar.WEEK_OF_YEAR;
+            case "day":
+            default:
```

Comment:
Is it possible to move the `default` at the bottom of the `switch` ?

Reply:
Sure! I've moved it to the bottom now.